runtime/ops/worker_host.rs

```diff
@@ -451,7 +454,7 @@ where
     granted_list: value
       .paths
       .into_iter()
-      .map(|env| EnvDescriptor(env.to_uppercase()))
```

Comment:
Seems like a bug? I think this `.to_uppercase()` should only happen on Windows like the rest of the code does.

Reply:
I believe you are correct